alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/accessor/ProviderDataAccessor.java

```diff
@@ -50,4 +51,6 @@
     List<ProviderUserModel> getAllUsers(final String providerName);
 
     List<ProviderUserModel> deleteAndSaveAllUsers(final String providerName, final Collection<ProviderUserModel> usersToDelete, final Collection<ProviderUserModel> usersToAdd);
+
+    void updateProjectAndUserData(String providerName, final Map<ProviderProject, Set<String>> projectToUserData);
```

Comment:
Can we rename the `projectToUserData` variable to `projectToUserEmailMap` or something that clarifies what the Set of Strings is?

Reply:
We could but I was thinking that if we change that to a User object in the future, because it becomes more than just email addresses, then we wouldn't have to change the method name too.